### ISSUE TITLE
Gaps in Distro Testing

### ISSUE THREAD

**Issue by superterran** [glibg10b/ltt-linux-challenge-issues#15]
One thing that is not apparent with your recent Linux reviews is that Linux offers the best desktop experiences out of all the major players. 

To showcase this, I suggest trying the following distros:

# Fedora 35

Fedora offers a vanilla version of GNOME 41, makes it easy to install Steam and graphics drivers, and provides the latest technologies that most distros aren't ready to integrate (pulsewire, wayland, etc). It's my opinion that this distro represents the best Linux desktop environment for normal users. 

GNOME 41 is a very interesting desktop, and any review of it should account for the learning curve involved to understand its expected user flow. The Overview, while unpopular to those new to it, is well-considered and is a direct evolution of macOS concepts. 

Consider Apple's Mission Control, and compare it to the GNOME implementation which it calls the Activities Overview:

![MacOS Mission Control](https://user-images.githubusercontent.com/2163715/144769088-26615c2f-cfa5-4dd1-b899-f9a0d9739442.png)

The MacOS Mission Control shows the Dock on the bottom (with an App launcher by default), shows Windows in Expose, and a view of the virtual desktops.

![GNOME 41 Overview](https://user-images.githubusercontent.com/2163715/144769114-f8560abc-279d-4fb9-b7f8-e41124be6951.png)

GNOME provides all of this too, along with an inline search tool (what MacOS calls Spotlight), all at the press of a key. It can also be activated by a hot corner,  touchpad gesture, or top-left button. On Mac, these are several different services without a unified interface or keybindings. GNOME can showcase all of this functionality to new users by simply showing the Overview on login. 

# KDE Neon

The folks at KDE maintain its own Ubuntu-based distribution where they provide packages directly from their CI/CD pipes. This means that this version of KDE is exactly as the maintainers intended, and doesn't have any distro-level changes and themes which usually only drag the experience down. Newer versions of KDE Plasma are gorgeous and extremely easy to use. While the underlining distro isn't a bleeding edge as Fedora, it's based on Linux and is very easy to install Steam and other packages.

![image](https://user-images.githubusercontent.com/2163715/144769894-c5dfb275-c097-4080-8731-33ab6479a677.png)

KDE Plasma compares to Windows 11 in terms of visual design and customization. Windows and Mac still offer more stable, easier-to-use environments, but it's becoming hard to argue that either Desktop Environment matches their open-source rivals in terms of general user experience. KDE specifically offers native blurring effects that provide a very attractive user experience reminiscent of other major players. It also subtly supports several approaches to desktop and window management, making it great for users who don't want to adopt to something new like GNOME. Plasma allows you to build almost any user interface, it's very powerful.

**Comment by PreyK** [comment]
“Linux offers the best desktop experiences out of all the major players”
What do you mean?
This is just straight out not true.

Gnome is locked down and stubborn (like what’s the point of the top bar is it has no global menu? It’s just wasted space, you need to install separate tweaks to have basic options that should be there by default etc..)

And KDE is just all over the place.

if you want the best desktop experience you’d just buy a mac.

**Comment by superterran** [comment]
> “Linux offers the best desktop experiences out of all the major players”
> What do you mean?
> This is just straight out not true.

Above I compared Mission Control to the Activities Overview, bits of functionality and user-flow that's practically identical between the two platforms. But consider the differences, GNOME has improved on the implementation In the following ways:

* Windows in Expose have a close button, in Mission Control you must select a window - leave the interface -and then close it
* System Search is available in the overview, it's a different flow entirely with Apple's Mission Control
* Overview provides a visual tour of all functionality, there is no unified interface in MacOS to access apps, windows, search and system management menus... In this way, GNOME improves on MacOS's usability. It would be easier to introduce a new user to GNOME than it would be to introduce them to MacOS

> Gnome is locked down and stubborn (like what’s the point of the top bar is it has no global menu? It’s just wasted space, you need to install separate tweaks to have basic options that should be there by default etc..)

These are subjective complaints, and while I understand you don't like the interface, others (myself included) _deeply_ enjoy the GNOME interface. For those who do use the interface regularly, I believe the consensus is that parts of the interface are in fact easier and better considered than a Mac.

> if you want the best desktop experience you’d just buy a mac.

This is true for the hardware, it's basically unreasonable at this point to consider anything other than Apple hardware. But as a longtime Linux user who has a deep familiarity with both platforms, my biggest pause about making the switch is literally abandoning GNOME for MacOS. I actually spent the weekend using a 2020 Macbook to see how I'd adapt to the interface, and it was quickly apparent that MacOS didn't spark the same joy GNOME does.
